README.md

```diff
@@ -57,9 +57,9 @@ Note that the external-provisioner does not scale with more replicas. Only one e
 
 * `--worker-threads <num>`: Number of simultaneously running `ControllerCreateVolume` and `ControllerDeleteVolume` operations. Default value is `100`.
 
-* `--kube-api-qps <num>`: QPS for clients that communicate with the kubernetes apiserver. Defaults to `5.0`.
+* `--kube-api-qps <num>`: [QPS](https://github.com/kubernetes/client-go/blob/master/rest/config.go#L115) for clients that communicate with the Kubernetes apiserver. Defaults to `5.0`.
```

Comment:
Can you point to code in a specific release?  Otherwise the line of code in the master branch could change any time.

Reply:
good point